app/views/notifier/order_status_changed_to_approved.html.haml

```diff
@@ -1,16 +1,18 @@
 %h1= @order_detail.facility
 
-%p The following reservation has been approved.
+%p= text("header.#{@order_detail.product.class.name.downcase}")
```

Comment:
Not a big deal, but to future proof against multi-word classes, would `underscore` be better?